.github/workflows/main.yml

```diff
@@ -112,7 +122,78 @@ jobs:
+
+      # Java is needed for building the APK, see
+      # https://github.com/marketplace/actions/flutter-action.
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Set Flutter version from FVM config file to environment variables
+        uses: kuhnroyal/flutter-fvm-config-action@v1
+
+      - uses: subosito/flutter-action@main
+        with:
+          flutter-version: ${{ env.FLUTTER_VERSION }}
+          channel: ${{ env.FLUTTER_CHANNEL }}
+      
+      - name: Run integration tests
+        uses: reactivecircus/android-emulator-runner@v2
+        env:
+          USER_1_EMAIL: ${{ secrets.INTEGRATION_TEST_USER_1_EMAIL }}
+          USER_1_PASSWORD: ${{ secrets.INTEGRATION_TEST_USER_1_PASSWORD }}
+        # We are using the "google_atd" ("Automated Test Device") target because it's designed for
+        # automated testing and thus uses fewer resources.
+        #
+        # See "Slimmer Emulator System Images" / "Automated Test Device" at:
+        # https://android-developers.googleblog.com/2021/10/whats-new-in-scalable-automated-testing.html.
+        #
+        # Because of the use of "google_atd" we are bit limited in the choice of
+        # API level, arch and channel. See here all requirements:
+        # https://github.com/ReactiveCircus/android-emulator-runner#configurations 
+        # Additionally taking Screenshots does not work on ATD (see the google blog link further above).
+        with:
+          api-level: 30
+          target: google_atd
+          arch: x86
+          profile: pixel_3a
+          channel: canary
+          # We use some boot options for the following reasons:
+          # * "-no-boot-anim": Disables the boot animation for faster booting.
+          # * "-no-window": Disables the graphical windows, so no display is
+          #   required.
+          # * "-no-snapshot": Disables the quick boot feature. Therefore, the
+          #   emulator does not load or save the emulator state. We want to have
+          #   a fresh testing environment for every test run.
+          # * "-camera-back virtualscene": Shows this virtual room when opening
+          #   the back camera where you can walk. We are using this option
+          #   because most emulator have this option enabled when you create you
+          #   emulator via Android Studio. Can be helpful for testing things
+          #   with the camera. It's also possible to inject images into the
+          #   virtual scene to test things like qr code scanner.
+          # * "-camera-front": Shows this green monster when opening the front
+          #   camera. We are using this option because most emulator have this
+          #   option enabled when you create your emulator via Android Studio.
+          #   Can be helpful to test things with the camera.
+          emulator-options: -no-snapshot -no-window -no-boot-anim -camera-back virtualscene -camera-front emulated
```

Comment:
Would it also work without these? Do have to specify them? If we have no use for them right now I would leave them out.

Reply:
It should also work without the camera parameter 👍 

But we need to remember to set these again when we need these (or assume this person has this knowledge) because as I mentioned has the Android Emulators created with Android Studio by default these options. When I implemented the integration tests for the qr code scanner took me this a few hours to figure out why the integration tests are working locally but no with GitHub Actions.

I will remove the options 👍

Reply:
Ah I misunderstood your comments then. Please re-add them (sorry!) but change them a bit.   

Instead of 
> Can be helpful for testing things with the camera.

Write : Is **needed** to test things with the camera

And instead of 
> We are using this option because most emulator have this option enabled when you create you emulator via Android Studio.

Write something along the lines of: Default option for emulators created by Android Studio. We use this so we have the same environment locally as when using this action.

Does this make sense?